node/node_handler.go

```diff
@@ -290,6 +290,16 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block) {
 	node.UpdateStakingList(node.QueryStakeInfo())
 	node.printStakingList()
 	node.blockchain.StoreNewShardState(newBlock, &node.CurrentStakes)
+
+	node.Debug()
+}
+
+// Debug --
+func (node *Node) Debug() {
+	players, balances := node.GetResult()
+	for i := range players {
+		utils.GetLogInstance().Info("******RESULT****", "player", players[i], "balance", balances[i])
```

Comment:
INFO [03-15|22:30:53.355] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x1a3e7a44ee21101d7D64FBf29B0F6F1fc295F723 balance=3875820019684212736
INFO [03-15|22:30:53.355] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x10A02A0a6e95a676AE23e2db04BEa3D1B8b7ca2E balance=3875820019684212736
INFO [03-15|22:30:53.355] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x3e881F6C36A3A14a2D1816b0A5471d1caBB16F33 balance=3875820019684212736
INFO [03-15|22:30:53.355] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x9d72989b68777a1f3FfD6F1DB079f1928373eE52 balance=3875820019684212736
INFO [03-15|22:30:53.355] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x67957240b6eB045E17B47dcE98102f09aaC03435 balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0xf70fBDB1AD002baDF19024785b1a4bf6F841F558 balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x3f1A559be93C9456Ca75712535Fd522f5EC22c6B balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0xedD257B4e0F5e7d632c737f4277e93b64DC268FC balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x66A74477FC1dd0F4924ed943C1d2F1Dece3Ab138 balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x04178CdbCe3a9Ff9Ea385777aFc4b78B3E745281 balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x46C61d50874A7A06D29FF89a710AbBD0856265be balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0xfE9BABE6904C28E31971337738FBCBAF8c72873e balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0x3f78622de8D8f87EAa0E8b28C2851e2450E91250 balance=3875820019684212736
INFO [03-15|22:30:53.356] ******RESULT****                         port=9000 ip=127.0.0.1 player=0xd2Cb501B40D3a9a013A38267a4d2A4Cf6bD2CAa8 balance=3875820019684212736

where each balance is not 1000 ether